### PR TITLE
Scrum 7 delete rate button

### PR DESCRIPTION
**Pull request by SneguRoma**
1. Created deleteRateButton component.
2. Refactored rate and rates component: constants removed to constants.ts,  created interfaces to constants and added to interfaces.ts
3. Added deleteRateButton to Rates and rate components.

### CODE REVIEW

File: src/Constants.tsx
Comment (Bogdansavel) [inline src/Constants.tsx:1]: don't forget to comment values for local testing and uncomment production values

File: src/Constants.tsx [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export const BASE_URL = "https://kk-backend-619198175847.europe-central2.run.app"
```
Comment (Bogdansavel) [inline src/Constants.tsx]: don't forget to comment values for local testing and uncomment production values

File: src/components/App.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import Rate from './Rate';
+import Rate from './Rate/Rate';
 import RegisteredMembers from './RegisteredMembers';
 import { Routes, Route } from "react-router-dom";
-import Rates from './Rates';
+import Rates from './Rates/Rates';
```
Comment (Bogdansavel) [inline src/components/App.tsx]: why do you need folder for each and every component?

File: src/components/Rate/Rate.tsx [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+  defaultRate,
+  mockUser,
+} from "../constants";
+import DeleteRateButton from "../DeleteRateButton/DeleteRateButton";
+import { IRate, IMovie } from "../interfaces";
+
+function Rate() {
+  const { movieId } = useParams();
+  const { webApp, executeMethod } = useTelegram();
+  /* const { id, username, first_name } = webApp.initDataUnsafe?.user */
```
Comment (Bogdansavel) [inline src/components/Rate/Rate.tsx]: don't forget to comment values for local testing and uncomment production values

File: src/components/Rates/Rates.tsx [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+import { defaultAverage, defaultMovie, defaultYourRate } from "../constants";
+import { IMovie, IAverage, IYourRate } from "../interfaces";
+import DeleteRateButton from "../DeleteRateButton/DeleteRateButton";
+
+function Rates() {
+  const { movieId } = useParams();
+  const navigate = useNavigate();
+  const { webApp, executeMethod } = useTelegram();
+  //const { username } = webApp.initDataUnsafe?.user
+  const username = "test";
```
Comment (Bogdansavel) [inline src/components/Rates/Rates.tsx]: don't forget to comment values for local testing and uncomment production values

**Comment by Bogdansavel** [comment]
- Make DeleteButton component with solid red background (not gradient) and same color as heart icon. 
- Make it darker on hover. 
- Make text of the button change color due to theme using telegram-text css class

File: src/components/DeleteRateButton/DeleteRateButton.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  };
+
+  return (
+    <Box className="flex justify-center">
+      <Button
+        gradientMonochrome="failure"
+        onClick={handleDelete}
+        className="px-1 mx-6 my-3"
+      >
+        Удалить
```
Comment (Bogdansavel) [inline src/components/DeleteRateButton/DeleteRateButton.tsx]: change please to "Удалить оценку"

File: src/components/App.tsx (hunk 2 above)
Comment (SneguRoma) [reply src/components/App.tsx]: Usually in React (how I learned in my school ) ) files of components grouped  in own directories with additional files like as constants, styles and etc. But I checked  React  docs and I think you are right in our case we have a not big app and this schema without own directories or file organization is good

**Comment by Bogdansavel** [comment]
After deletion of rate have to update average rate as well

**Pull request merged**